modules/serverless-cluster-sizing-req.adoc

```diff
@@ -1,27 +0,0 @@
-// Module included in the following assemblies:
-//
-// * /serverless/admin_guide/install-serverless-operator.adoc
-
-[id="serverless-cluster-sizing-req_{context}"]
-= Defining cluster size requirements for an {ServerlessProductName} installation
-
-To install and use {ServerlessProductName}, the {product-title} cluster must be sized correctly.
-
-[NOTE]
-====
-The following requirements relate only to the pool of worker machines of the {product-title} cluster. Control plane nodes are not used for general scheduling and are omitted from the requirements.
-====
-
-The minimum requirement for {ServerlessProductName} is a cluster with 10 CPUs and 40GB memory.
-
-The total size requirements to run {ServerlessProductName} are dependent on the applications deployed. By default, each pod requests approximately 400m of CPU, so the minimum requirements are based on this value.
-
-In the size requirement provided, an application can scale up to 10 replicas. Lowering the actual CPU request of applications can increase the number of possible replicas.
-
-[NOTE]
-====
-The following limitations apply to all {ServerlessProductName} deployments:
-
-* Maximum number of Knative services: 1000
-* Maximum number of Knative revisions: 1000
-====
```

Comment:
I wonder if we should leave those. This is not minimal requirements, but something we've checked is okay to support.